docs/zh/start/rpc/dubbo_json_rpc.md

```diff
@@ -15,21 +15,16 @@ go build -o layotto cmd/layotto/main.go
 
 这里使用了`dubbo-go-samples`提供的示例服务
 ```sh
-git clone git@github.com:apache/dubbo-go-samples.git
+git clone git@github.com:apache/dubbo-go-samples.git (f0d1e1076397a4736de080ffb16cd0963c8c2f9d since Sun Mar 20 22:01:12)
```

Comment:
Emmmmm.....
Users can't just copy-paste and run this command with this ` (f0d1e1076397a4736de080ffb16cd0963c8c2f9d since Sun Mar 20 22:01:12)`. 

Maybe we can:
```shell
git clone https://github.com/apache/dubbo-go-samples.git
cd dubbo-go-samples
git reset --hard f0d1e1076397a4736de080ffb16cd0963c8c2f9d
```
So that users can just copy-paste and run these commands.
What do you think?

Reply:
I just deleted the commit hash, because this head is the latest master. If there are any problems later, I will fix it.